.github/workflows/ci.yml

```diff
@@ -32,7 +34,7 @@ jobs:
       run: rustup set profile minimal
     - uses: arduino/setup-protoc@v3
     - name: Test
-      run: ./scripts/test.sh
+      run: bash ./scripts/test.sh
```

Comment:
Is there some scenario where the `bash` prefix is required?

Reply:
`bash` prefix lets us run `.sh` files on Windows.